bc_obps/registration/fixtures/mock/user.json

```diff
@@ -5,7 +5,7 @@
     "fields": {
       "user_guid": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
       "business_guid": "efb76d57-88b7-4eb6-9f26-ec12b49c14c1",
-      "bceid_business_name": "Example Business",
+      "bceid_business_name": "New Operator 3 Legal Name",
```

Comment:
I only changed a couple of these because I think it's useful to see that the bceid business name can be different from the legal name (suspicious suspicious)